src/clientmetricreport/DefaultClientMetricReport.ts

```diff
@@ -116,28 +134,19 @@ export default class DefaultClientMetricReport implements ClientMetricReport {
       source?: string;
     };
   } = {
-    googActualEncBitrate: {
-      transform: this.identityValue,
-      type: SdkMetric.Type.VIDEO_ACTUAL_ENCODER_BITRATE,
-    },
-    googAvailableSendBandwidth: {
-      transform: this.identityValue,
-      type: SdkMetric.Type.VIDEO_AVAILABLE_SEND_BANDWIDTH,
-    },
-    googRetransmitBitrate: {
+    retransmittedBytesSent: {
       transform: this.identityValue,
       type: SdkMetric.Type.VIDEO_RETRANSMIT_BITRATE,
     },
-    googAvailableReceiveBandwidth: {
+    totalEncodedBytesTarget: {
       transform: this.identityValue,
-      type: SdkMetric.Type.VIDEO_AVAILABLE_RECEIVE_BANDWIDTH,
+      type: SdkMetric.Type.VIDEO_TARGET_ENCODER_BITRATE,
     },
-    googTargetEncBitrate: {
+    totalPacketSendDelay: {
       transform: this.identityValue,
-      type: SdkMetric.Type.VIDEO_TARGET_ENCODER_BITRATE,
+      type: SdkMetric.Type.VIDEO_BUCKET_DELAY_MS,
     },
-    googBucketDelay: { transform: this.identityValue, type: SdkMetric.Type.VIDEO_BUCKET_DELAY_MS },
-    googRtt: { transform: this.identityValue, type: SdkMetric.Type.STUN_RTT_MS },
+    roundTripTime: { transform: this.identityValue, type: SdkMetric.Type.STUN_RTT_MS },
```

Comment:
Is this needed?

Reply:
This one is duplicated, removed already.